src/promptflow/tests/sdk_cli_test/e2etests/test_flow_as_func.py

```diff
@@ -33,6 +33,23 @@ def test_flow_as_a_func(self):
         assert result["output"] is None
         assert "line_number" not in result
 
+    @pytest.mark.asyncio
+    async def test_flow_as_a_func_asynckw(self):
+        f = load_flow(f"{FLOWS_DIR}/print_env_var", async_call=True)
+        result = await f(key="PATH")
+        assert result["output"] is not None
+
+    @pytest.mark.asyncio
+    async def test_async_flow_as_a_func_asynckw(self):
+        f = load_flow(f"{FLOWS_DIR}/print_env_var_async", async_call=True)
+        result = await f(key="PATH")
+        assert result["output"] is not None
+
+    def test_async_flow_as_a_func_wo_asynckw(self):
+        f = load_flow(f"{FLOWS_DIR}/print_env_var_async")
```

Comment:
do we define async flow as dag flow that use async tools?

Then could we run flow with sync tools in async manner？

Reply:
Yes, the first test is sync tool run in async manner.